packages/beacon-node/src/db/repositories/attesterSlashing.ts

```diff
@@ -1,18 +1,26 @@
 import {ChainForkConfig} from "@lodestar/config";
 import {Db, Repository} from "@lodestar/db";
-import {ValidatorIndex, phase0, ssz} from "@lodestar/types";
+import {ForkName, isForkPostElectra} from "@lodestar/params";
+import {AttesterSlashing, ValidatorIndex, phase0, ssz, sszTypesFor} from "@lodestar/types";
 import {Bucket, getBucketNameByValue} from "../buckets.js";
 
+// We add a 1-byte prefix where 0 means pre-electra and 1 means post-electra
```

Comment:
Do we really need this? It should be sufficient to just use `ssz.electra.AttesterSlashing` to serialize / deserialize data, we already validate on api and gossip (using proper ssz type) that the limit is correct

Reply:
You are right. ssz.phase0.AttesterSlashing and ssz.electra.AttesterSlashing should make no difference when it comes to serializing and deserializing. Only difference is the length